### DISCUSSION TITLE
Video upload recommendations?

### DISCUSSION THREAD

**Discussion by BackwardEcho** [aegean-odyssey/mpmd_marlin_1.1.x#97] [General]
I recorded myself soldering the components [for the new fan output](https://github.com/aegean-odyssey/mpmd_marlin_1.1.x/wiki/Extra-Fan-Output) to my original stock controller board and testing it out. Both videos are a couple hundred megabytes in size, how best would you recommend I upload them to the github wiki on the topic?

**Comment by BackwardEcho** [comment]
OK.... after editing the video of the actual soldering the file size is now 826 MB. BUT. It is 32 minutes long with really no jump cuts except when I needed to find a working desoldering gun and use it.

**Comment by PurpleHullPeas** [comment]
YouTube, and then link it wherever you please. You could set the video to "unlisted" with comments disabled if you don't want to deal with traffic outside of GitHub.

**Reply by BackwardEcho** [reply]
@PurpleHullPeas, I would much rather upload it somewhere that didn't make it a big deal to upload a video. While Google Drive is something I have used before for MPMD stuff, like my bed switch mod, I feel like this type of video would be better suited to upload directly to this github if it were possible. Also, I am generally hoping to upload it to someplace that doesn't care about the total space used (like how Google Drive only gives you 15 GB free) because it IS going be about one gigabyte in total size with the two videos.

As for YouTube, I was going through the process just now until it said I had to verify my phone number to upload videos longer than 15 minutes. I don't really like YouTube / Google's company practices already, but at least directly uploading it to my Google Drive means YouTube wont be able to pull any BS with me while I upload a very niche video for a very niche community.

**Reply by BackwardEcho** [reply]
I went and verified it >_>

**Comment by BackwardEcho** [comment]
[Well, here it is](https://youtube.com/playlist?list=PLI6F9f73tlcTNmQx5lNf-VmxEcKDCImxT)

Should I make an edit to something?
